src/core/defaults.json

```diff
@@ -2,34 +2,29 @@
   "organisation": "UK Makerspace",
   "software-name": "Membership System",
   "logo": "",
-
   "home-text": "Hello there Makers!",
   "home-link-text": "Find out more",
   "home-link-url": "http://example.com/about",
   "home-redirect-url": "/login",
-
   "user-home-url": "/profile",
   "admin-home-url": "/admin",
   "referral-short-url": "http://example.com/refer",
-
   "support-email": "membership@example.com",
   "support-email-from": "UK Makerspace Membership",
-
   "contribution-min-monthly-amount": "1",
+  "tax-rate-enabled": "false",
+  "tax-rate-percentage": "7",
```

Comment:
I can't see where this is set? It looks like it's used in `/content/payment` but not set anywhere. Possibly `GetContentPaymentDto.taxRate` should be fetched from Stripe?

Reply:
It's used here:

src/api/transformers/ContentTransformer.ts
```ts
  payment: withValue<"payment">({
    stripePublicKey: ["readonly", () => config.stripe.publicKey],
    stripeCountry: ["readonly", () => config.stripe.country],
    taxRateEnabled: ["option", "tax-rate-enabled", "bool"],
    taxRate: ["option", "tax-rate-percentage", "int"]
  })
  ```
  
  as the default value if it's undefined in the database.

Reply:
Ahh I see, the value falls through to the `ContentTransformer.update` handler on PATCH. Sorry I missed that :+1: